### ISSUE TITLE
better error messages

### ISSUE THREAD

**Issue by hscells** [ielab/ir-runs-leaderboard#2]
for example, if the run file is invalid. there should be some validation on the run files when they are uploaded.

**Comment by guidozuc** [comment]
you can have a look at the script TREC distributes to check for validity of submission (may be in TREC Deep Learning Track webpage), or wait on output of trec_eval to check it is a valid output

**Comment by hanglics** [comment]
Do not have access to this repo, cannot push, now parked.
